packages/react/src/components/Slider/Slider.tsx

```diff
@@ -393,16 +544,18 @@ export default class Slider extends PureComponent {
     // If shift was held, account for the stepMultiplier
     if (evt.shiftKey) {
       const stepMultiplier = this.props.stepMultiplier;
-      delta *= stepMultiplier;
+      delta *= stepMultiplier ?? 1;
```

Comment:
This shouldn't be needed. `stepMultiplier` has a default of `4` set already. Using `1` if `stepMultiplier` is null seems unnecessary and could cause unwanted behavior.

Reply:
Yep, forgot about the defaults.